### DISCUSSION TITLE
Feature Request: Subtitle override

### DISCUSSION THREAD

**Discussion by BrandoCommando** [Shopify/function-examples#262] [Feedback: Cart Transform API]
Currently the merge operation supports a "title" parameter which can override the product title, which is nice. But the selected options line still shows (redundantly). It would be nice to be able to override that as well. Here's what I mean:
<img width="449" alt="image" src="https://github.com/Shopify/function-examples/assets/97893/3bbd9b40-88f8-4733-84db-3935e8f4534a">

I just want to remove that line, since it already shows under the child line item.

**Comment by gil--** [comment]
Would love to see this as well. Subtitle/variant title override now that the function supports overriding product name,
